### PR TITLE
Dev 2

### PR DESCRIPTION
**Pull request by MaximeRaguenes**
## Purpose
_Describe the problem or feature in addition to a link to the issues._

## Approach
_How does this change address the problem?_

#### Open Questions and Pre-Merge TODOs
- [ ] Use github checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

#### Blog Posts
- [How to Pull Request](https://github.com/flexyford/pull-request) Github Repo with Learning focused Pull Request Template.

## TO VALID PR

* All commits, vars, documentation, code are in English
* 1 feature = 1 PR, No more
* The code builds with Travis CI, and nothing looks unfinished
* When code is complicated, that’s documented, otherwise ask for comment 
* No node_modules is present
* Code respect [Airbnb style](https://hackernoon.com/configure-eslint-prettier-and-flow-in-vs-code-for-react-development-c9d95db07213) checked with Travis CI
* README explain Clearly how to run the project, otherwise reject the PR
* Chrome LightHouse CI score is upper than 80%

### CODE REVIEW

**Review by Menthx — APPROVED** [review]
Pas trop de fichiers, tout va bien #O

File: src/app/home/home.component.css [hunk 1]
```diff
@@ -0,0 +1 @@
+/* PD DE TRAVIS */
```
Comment (riderx) [inline src/app/home/home.component.css]: WTF, are you serious  ?

**Review by riderx — DISMISSED** [review]
remove empty file otherwise that good !

**Review by Menthx — APPROVED** [review]

**Review by BenbachirAzzedine — APPROVED** [review]
Good guy's

**Review by Jo-zTk — APPROVED** [review]
Beaucoup de code, un peu comme tout le monde cette semaine, 
Le lien ghpages fonctionne, j'ai survolé le code, mais pas tout vérifié.

**Comment by Mateo-M** [comment]
All good

**Comment by Mateo-M** [comment]
All good !

**Review by mateo-m — APPROVED** [review]
All good

**Review by yohannmartinez — APPROVED** [review]
cool

**Pull request merged**